src/assets/source/scripts/modules/serlo_side_navigation.js

```diff
@@ -287,6 +289,10 @@ define("side_navigation", ["jquery", "underscore", "referrer_history", "events",
                 $ul.append(menuItem.render().$el);
             });
 
+            if (self.levels[0][0].data.parent.data.community) {
```

Comment:
Yes, it works. `self.levels` contains all levels starting at the dropdown level (i.e. the main-menu does not appear there). So what I basically do, is to grab the first link in the first lvl and check whether its parent (i.e. the main-menu item) is a community link. When you go one level deeper, `self.levels` just adds one entry for the new level, but `self.levels[0]` remains unchanged. So there should be no problem when the dropdown contains at least one item (otherwise it wouldn't be shown anyway)